docs/docs/dev/contribute/introduction.md

```diff
@@ -0,0 +1,55 @@
+# Introduction
+
+There are multiple ways to make an impact in Kuma
+
+## Community
+
+You can join the slack channel or the community meetings as shown in the [community section](/community).
+
+## Documentation
+
+You can edit this documentation. To do so checkout [the getting started doc](https://github.com/kumahq/kuma-website/blob/master/CONTRIBUTING.md).
+
+## Core Code
+
+Most of Kuma is written in Go. Checkout the [contributing](https://github.com/kumahq/kuma/blob/master/CONTRIBUTING.md) documentation on how to get started.
+
+## GUI Code
+
+The UI is written in Vuejs and Checkout the [repo](https://github.com/kumahq/kuma-gui/blob/master/DEVELOPER.md) to see how to contribute.
+
+## Testing unreleased versions
+
+Kuma publishes new binaries for every commit.
+There's a small script to download the latest version:
+
+```shell
+curl https://kuma.io/cd.sh | sh -
```

Comment:
"cd" feels confusing here, too much connotation with the bash cd IMO. Maybe just "dev" or "preview"?

Reply:
I'll use preview then